config/manager/manager.yaml

```diff
@@ -28,6 +28,9 @@ spec:
         resources: {}
```

Comment:
No `runAsNonRoot` for the pod?

Reply:
Added it here instead - https://github.com/openshift/elasticsearch-operator/pull/917/files#diff-7263fc341f1b59afdc8599e1f570cdd180c28bf8cca71ecee1e58b58131202c5R43. Thats the same pod, but a different container.  The resulting CSV created has the `runAsNonRoot` here - https://github.com/openshift/elasticsearch-operator/pull/917/files#diff-7263fc341f1b59afdc8599e1f570cdd180c28bf8cca71ecee1e58b58131202c5R43

Reply:
Hmm, in that case. I would say maybe let's put that one for the pod in `manager` vs the `proxy_patch`.

Reply:
Done